automol/form/_form.py

```diff
@@ -1,13 +1,13 @@
-""" Library for dealing with molecular formula,
-    represented as dict[atom symbol: atom number]
+"""Library for dealing with molecular formula,
+represented as dict[atom symbol: atom number].
```

Comment:
Let's change this to "represented as a dictionary mapping atom symbols onto their stoichiometries." since the syntax here is not actual Python typing syntax.